app/app.go

```diff
@@ -29,6 +33,10 @@ import (
 	"github.com/cosmos/cosmos-sdk/version"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+
+	aclmodule "github.com/cosmos/cosmos-sdk/x/accesscontrol"
+	aclkeeper "github.com/cosmos/cosmos-sdk/x/accesscontrol/keeper"
+	acltypes "github.com/cosmos/cosmos-sdk/x/accesscontrol/types"
```

Comment:
This is for parallel TX:
`7248ade7e9e6f0b19498f55c758b27cdd11a0d1e`

Reply:
fixed